providers/africastalking/README.md

```diff
@@ -0,0 +1,15 @@
+# Novu Africastalking Provider
+
+Africastalking SMS provider library for [@novu/stateless](https://github.com/novuhq/novu)
+
+## Usage
+
+```javascript
+import { AfricastalkingSmsProvider } from '@novu/africastalking';
+
+const provider = new AfricastalkingSmsProvider({
+  apiKey: process.env.AT_API_KEY,
+  username: process.env.AT_USERNAME,
+  from: process.env.AT_SENDER_ID,
```

Comment:
```suggestion
  apiKey: process.env.AFRICASTALKING_API_KEY,
  username: process.env.AFRICASTALKING_USERNAME,
  from: process.env.AFRICASTALKING_SENDER_ID,
```
Let's be more verbose as it will help to avoid to clashes with other potential environment variables for configuring other providers.